lagtraj/input_definitions/examples_files/forcings_conversion/dephy_inversion_prescribed_rad.yaml

```
@@ -0,0 +1,36 @@
+levels_method   : copy
+version         : 1.0.0
+export_format   : dephy
+comment         : Forcing and initial conditions for Lagrangian case
+campaign        : EUREC4A
+source_domain   : n/a
+reference       : n/a
+author          : s.j. boeing
+modifications   : First version
+case            : n/a
+adv_temp        : 1
+adv_theta       : 1
+adv_thetal      : 1
+adv_qv          : 1
+adv_qt          : 1
+adv_rv          : 1
+adv_rt          : 1
+rad_temp        : 1
+rad_theta       : 1
+rad_thetal      : 1
+forc_omega      : 1
+forc_w          : 1
+forc_geo        : 1
+surfaceType     : ocean
+surfaceForcing  : ts
+surfaceForcingWind                        : z0_traj
+nudging_method_scalars                    : runtime_inversion_height
+nudging_transition_thickness_scalars      : 500
```

since these are examples might it be worthwhile to add inline comments for the units, e.g. 
```suggestion
nudging_transition_thickness_scalars      : 500  # [m]
```